lib/kubernetes-deploy/kubernetes_resource/pod_disruption_budget.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class PodDisruptionBudget < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
It's true that 30s is overly generous for both. In practice configmaps appear pretty much instantly (unless of course `apply` fails, in which case we'd already fail the deploy right away), but that's not guaranteed. How about 10s, which will lead us to look for it 3 times?